include/freerdp/freerdp.h

```diff
@@ -339,8 +343,13 @@ extern "C"
 
 		ALIGN64 rdpStreamDumpContext* dump; /* 64 */
 		ALIGN64 wLog* log;                  /* 65 */
+#if defined(WITH_SSO_MIB)
+		ALIGN64 MIBClientApp* mibClientApp; /* 66 */
```

Comment:
might be better off in `rdpClientContext`

Reply:
Is the `rdpContext` member of `freerdp` always a `rdpClientContext` inside `client/common/*`?

Reply:
Moved to `rdpClientContext` but the previous question still remains (I mean it works, but that's not a perfect indicator 😉 )

Reply:
see my comment below regarding wrapping the type in a private struct.